lms/djangoapps/course_home_api/progress/v1/views.py

```diff
@@ -106,21 +109,15 @@ def get(self, request, *args, **kwargs):
             reset_masquerade_data=True
         )
 
-        user_timezone_locale = user_timezone_locale_prefs(request)
-        user_timezone = user_timezone_locale['user_timezone']
```

Comment:
Just curious, why remove this?

Reply:
I deleted this because I didn't see `user_timezone` being used, so I didn't think I needed to surface it to the MFE. Is there a reason to keep this here?